client/src/components/PatientsTable/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { Table, Typography, Input, message, Spin } from 'antd';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+
+const { Search } = Input;
+const { Title } = Typography;
+
+const PatientsTable = () => {
+  const [dataSource, setDataSource] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const history = useHistory();
+
+  const fetchPatients = async () => {
+    try {
+      const {
+        data: { data },
+      } = await axios.get(`/api/v1/patients`);
+      return data;
+    } catch {
+      return message.error('Sever error, please try again later');
+    }
+  };
+
+  const initPatientList = async () => {
+    setLoading(true);
+    const data = await fetchPatients();
+    setDataSource(data);
+    setLoading(false);
+  };
```

Comment:
This function is unnecessary because you can do all these things in `fetchPatients` function and at the `useEffect`